src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.TargetFrameworkInference.targets

```diff
@@ -83,7 +83,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Target Name="_CheckForUnsupportedTargetFramework" BeforeTargets="_CheckForInvalidConfigurationAndPlatform">
     <NETSdkError Condition="'$(_UnsupportedTargetFrameworkError)' == 'true'" 
```

Comment:
I'm going to merge this because it's a strict improvement. Happy to approve another PR to do an even nicer error message.